pkg/utils/jira.go

```diff
@@ -0,0 +1,125 @@
+package utils
```

Comment:
@Nikokolas3270 Thanks for moving JIRA client as an interface. 
Shall we move into `github.com/openshift/backplane-cli/pkg/jira` rather than keeping it in package util?

Reply:
As We discussed offline, we will move Jira package in a separate PR